bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,37 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
```

Comment:
The filename can be specified but it doesn't yield results on the console. Instead, it prompts to overwrite the existing file with the changes whereas `--stdin` writes output to `--stdout`.
Quoting the usage:
> elm-format --stdin  # Format input from stdin and write to stdout
Also see https://github.com/avh4/elm-format#usage for more details.